frontend/javascripts/oxalis/geometries/cube.ts

```diff
@@ -99,7 +100,9 @@ class Cube {
       vec(min[0], min[1], max[2]),
       vec(min[0], max[1], max[2]),
       vec(min[0], max[1], min[2]),
-    ];
+    ]);
+
+    // todo: clarify what the cross sections are and fix them probably?
```

Comment:
todo

Reply:
@daniel-wer you probably have some insight here :)

Reply:
As far as I know, the cross sections simulate that the bounding box is a cuboid although it is actually only a wireframe of a cuboid. This is important so that the bounding box can be seen whenever one of its *faces* is visible in one of the viewports and not only whenever one of its *edges* is visible.
This is done by having one cross section per viewport which is a simple rectangle with 4 edges and moving it in the respective 3rd dimension whenever the flycam position changes.
Currently, this doesn't seem to work on the deployed branch.
Hope this makes sense, otherwise let's have a quick call :)

Reply:
thank you, makes total sense :+1:

Reply:
should be fixed now :tada: